# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/tests/browser.ini
# Commit: 6a0dcd8f4e3e
# Full Hash: 6a0dcd8f4e3e35d27edba6a4fef795cf8174ac78
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2021-02-18 04:16:06
# Regressor Bug: 1692350
# File Overlap Count: 1
# Description:
#   Bug 1692350 - Implement a telemetry probe to collect page load data for documents that use lazyload r=emilio,chutten
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104912
# ==============================================================================

diff -r 6d7590bfd8d3 -r 6a0dcd8f4e3e layout/base/tests/browser.ini
--- a/layout/base/tests/browser.ini	Wed Feb 17 17:08:51 2021 +0000
+++ b/layout/base/tests/browser.ini	Wed Feb 17 17:17:52 2021 +0000
@@ -26,3 +26,7 @@
 support-files =
   file_lazyload_telemetry.html
   image_rgrg-256x256.png
+[browser_lazyload_page_load_telemetry_iframe.js]
+support-files =
+  file_lazyload_telemetry.html
+  image_rgrg-256x256.png